packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -1755,14 +1756,16 @@ describe("Editing", () => {
 				parentField: brand("foo"),
 				parentIndex: 0,
 			};
-			assert.throws(() =>
-				tree.editor.move(
-					{ parent: rootNode, field: brand("foo") },
-					0,
-					1,
-					{ parent: fooPath, field: brand("bar") },
-					0,
-				),
+			assert.throws(
+				() =>
+					tree.editor.move(
+						{ parent: rootNode, field: brand("foo") },
+						0,
+						1,
+						{ parent: fooPath, field: brand("bar") },
+						0,
+					),
```

Comment:
Consider using our validateUsageError uttil (its in src/test/simple-tree/utils.ts and would need to be moved to src/test/utils.ts) for more concise explicit validation.

Also if you want to test the entire error message, you can use a string not a regular expression, Just use a regex if you are matching part of the message,

Reply:
It complained when I used the string. Either way I'm using `validateUsageError` now. Thanks for pointing it out.